### DISCUSSION TITLE
Add Thai to meeting types [Feature Request]:

### DISCUSSION THREAD

**Discussion by joefincher** [code4recovery/12-step-meeting-list#1161] [Q&A] [answered]
### Contact Details

joefincher@yahoo.com

### Requested Feature/Enhancement

We are in the process of rebuilding aathailand.org in WordPress using TSML. We would like to request that  the Thai designator be added to meeting types so that it will be available on the Meeting Finder app as well as our website.

**Comment by joshreisner** [comment, marked as answer]
hi joe, we request that people open discussions, not issues. 

we have an [open pull request](https://github.com/code4recovery/spec/pull/70) to add several new types that the General Service Office is reviewing, i've added Thai to that request

in the meantime, you can add thai [as a custom type to your website](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21) with the following code:

```php
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'TH' => 'Thai',
    ));
}
```

**Reply by joshreisner** [reply]
(please be sure to use `TH` in the code so that when we add it to the plugin the tag will carry over)
